README.md

```diff
@@ -46,102 +46,94 @@ $ mvn install
 ### Search creative images with phrase, age of people, and page
 
 ```java
-        String apiKey = "API Key";
-        String apiSecret = "API Secret";
-        String userName = "Username";
-        String userPassword = "Password";
-
-        ApiClient client = ApiClient.GetApiClientWithResourceOwnerCredentials(apiKey, apiSecret, userName, userPassword);
-
-      try {
-            SearchImagesCreative search = client.searchimagescreative()
-                    .withPhrase("cat")
-                    .withAgeOfPeople(EnumSet.of(AgeOfPeople.CHILD, AgeOfPeople.BABY,AgeOfPeople.ADULT))
-                    .withPage(3);
-            String result = search.executeAsync();
-            System.out.print(result);
-
-        } catch (SdkException e) {
-            System.out.println("Exception occurred while searching for creative images: " + e.getLocalizedMessage());
-            System.exit(-1);
-        }
+String apiKey = "API Key";
+String apiSecret = "API Secret";
+
+ApiClient client = ApiClient.GetApiClientWithClientCredentials(apiKey, apiSecret);
```

Comment:
In a recent conversation I recall we wanted to discourage the use of resource owner grants.

Reply:
Let's check if anyone is using it before we remove it.